samples/notebooks/prediction/SDK_Custom_Predict.ipynb

```diff
@@ -861,6 +861,8 @@
     "\n",
     "First authorize Google auth library to access the Cloud Platform with Google user credentials. This step requires to put user inputs. If you are using a non-interactive shells, you should open a terminal to run this command. If you are using Vertex Workbench, you will need to open [a terminal](https://jupyterlab.readthedocs.io/en/stable/user/terminal.html) to run this command.\n",
     "\n",
+    "Note that if you are running this command on machines without a window manager/browser, it will prompt a command `gcloud auth application-default login --remote-bootstrap=\"...\"` which requires you to run on machines where you can launch a web browser, for example, a laptop, and paste the result to this command to continue the login flow. If you are running this command on machines with web browsers, it should have just worked and there is no need to use a separate machine to do it.\n",
```

Comment:
Reworded.

Note that if you are running the above command on a machine without a window manager/browser, it will prompt you to run the command `gcloud auth application-default login --remote-bootstrap=\"...\"`. Only machines that can launch web browsers, e.g. laptops or workstations, are allowed to run this command. After completing the in browser authentication steps, you'll need to copy and paste the verification code back into the original terminal to continue the login flow. If you are running this command on machines with web browser support, there is no need to use a separate machine.\n"

Reply:
Done.